public/stylesheets/cards.css

```diff
@@ -58,11 +58,11 @@ body {
   margin-bottom: 3px;
   display: block;
 }
-.field-input {
+.field-input,
+.text-input {
```

Comment:
Is there a reason to create a new class here?

Reply:
Yes, the .text-input fields are the ones used for login and register inputs. the .field-input class is further styled to make the button like inputs in play-settings and play pages. I guess it could be done somehow differently but this seemed like the simplest way to do it to me.